yarn-project/types/src/interfaces/pxe.ts

```diff
@@ -42,6 +42,12 @@ export interface PXE {
    */
   addAuthWitness(authWitness: AuthWitness): Promise<void>;
 
+  /**
+   * Adding a mint to the pez dispenser.
```

Comment:
Consider using the full form of PEZ - external folks look at it as docs
Also as @benesjan highlighted, add/popMint() could be changed to addTo/popFromSlowUpdatesTree()?

Not sure if `mint` is the right term here as well